.github/workflows/build-examples-gh-pages-on-push.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           pip install idf-component-manager --upgrade
 
       - name: Action for building binaries and config.toml
-        uses: espressif/idf-examples-launchpad-ci-action@v1.0
+        uses: espressif/idf-examples-launchpad-ci-action@v1.0.1
```

Comment:
Can we enable automatic updates to 1.0.x releases?

Reply:
If the updates are going to be released more frequently, I will create a PR and change the version of the action to the branch instead of the tag